aws-predictions/src/main/java/com/amplifyframework/predictions/aws/models/liveness/ChallengeEvent.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amplifyframework.predictions.aws.models.liveness
+
+import com.amplifyframework.predictions.models.ChallengeType
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+internal data class ChallengeEvent(
```

Comment:
Have you tested what happens when Liveness sends an event we can't parse into a ChallengeType we understand. Also still confused when ChallengeType and FaceLivenessChallengeType is used.

Reply:
Before, a new event would fail to get decoded and fail silently and the client app would get confused as to what to do (oops!). Added an exception that gets thrown in the websocket class when a challengetype fails to get parsed.

And from a previous comment reply, I've removed ChallengeType in favor of FaceLivenessChallengeType